NOAA-OWP/hydrovis
# Terraform Remote State

This update removes the old local Terraform state files and swaps out the Terraform backend for a remote state hosted in an S3 bucket on our Production account.

To test this simply do the following:

- Ensure that your `Core/sensitive` folder is up to date from Google Drive (or VLab in the future)
- Navigate to the `Core` folder
- `terraform init`
- `terraform workspace select ti`
- `terraform plan -out buildplan`
- Verify that the Terraform state is up to date

[SmartSheet Card](https://app.smartsheetgov.com/sheets/FwqPVjGh6Qwv9GWh8hG35rj2Rr3g7RFMF3jcj4h1?rowId=8696284002445188)
